docs/Cheatsheet.rst

```diff
@@ -19,8 +19,44 @@ which is the original source of information.
 Moreover, the official documentation is updated regularly, whereas the cheat sheet
 is probably going to be updated once a year (around April when the new DRAC allocations are announced).
 
+Comments and suggestions are welcome (`idt.cheatsheet@mila.quebec`).
+Please also signal errors if you spot them before we do.
+
+
 Errata
 ======
 
 Here is a list of the known errors in the cheat sheet that will have to be fixed in the next version.
 
+Partition preemption is not explained accurately on page 2
+----------------------------------------------------------
+
+The preemption on the Mila cluster is a bit more complicated than what is described in the cheat sheet.
+The jobs from the `long` partition can be preempted to allow jobs in the `main` partition to run,
+but jobs in `main` are never going to be preempted to allow for other jobs in `main`, no matter how much
+of the "fair use" a user has already consumed.
+
+This is why it can be considered rude (or just bad practice) to queue a large number of jobs
+on `main` at midnight on the Mila cluster with a lot of free capacity.
```

Comment:
It is not possible to run more than two `main` jobs in parallel per user, because of the requirement of 1+ GPU per GPU job (obvious) and maximum 2 GPUs used in total by jobs inside the `main` partition.

I'd stick to `long` for the explanation and say that `unkillable` jobs can preempt `main`, `main` can preempt `long`, but like never preempts like.

Reply:
Oh, wow, so that "max resource usage" is a total among all the jobs in that partition, and not just a per-job maximum ...

Then we're kind of back to square one in that people are going to need to submit the bulk of their stuff to `long` and hope that it runs there, and those resources are going to get hogged by someone who submitted 100 jobs yesterday when the cluster was idle.

It's just that everyone gets a 2 GPU from `main` in order to declog the cluster slowly, and those will preempt the jobs from whoever submitted the 100 jobs to `long`. Is that it?

Reply:
I've modified my description to be less thorough (and thus less wrong). I think it's more reasonable this way, and I will also open another ticket here to represent the fact that we should document this partition thing better on docs.mila.quebec.